fiftyone/core/context.py

```diff
@@ -140,6 +141,18 @@ def _get_context():
     return _context
 
 
+def _requires_http_polling():
+    ### SSE is unreliable in GitHub Codespaces
+    if os.environ.get("CODESPACES", None) == "true":
```

Comment:
how about:

```suggestion
    if os.environ.get("CODESPACES", "false").lower() == "true":
```

Reply:
`true` is precisely what we expect if we are in a GitHub Codepaces context. I think the above is sufficient